.github/workflows/ci_integration.yml

```diff
@@ -1,373 +1,354 @@
-        run: "poetry run invoke unittest-coverage"
-    needs:
-      - "pylint"
-      - "tests-postgres"
-      - "tests-mysql"
+  # integration-test:
+  #   runs-on: "ubuntu-22.04"
+  #   env:
+  #     INVOKE_NAUTOBOT_LOCAL: "True"
+  #     NAUTOBOT_DB_USER: "nautobot"
+  #     NAUTOBOT_DB_PASSWORD: "decinablesprewad"
+  #     NAUTOBOT_DB_ENGINE: "django_prometheus.db.backends.postgresql"
+  #     NAUTOBOT_SELENIUM_URL: "http://localhost:4444/wd/hub"
+  #   services:
+  #     postgres:
+  #       image: "postgres:13-alpine"
+  #       env:
+  #         POSTGRES_USER: "nautobot"
+  #         POSTGRES_PASSWORD: "decinablesprewad"
+  #         POSTGRES_DB: "nautobot"
+  #       ports:
+  #         - "5432:5432"
+  #     redis:
+  #       image: "redis:6-alpine"
+  #       ports:
+  #         - "6379:6379"
+  #     selenium:
+  #       image: "selenium/standalone-firefox:4.1.2"
+  #       ports:
+  #         - "4444:4444"
+  #   steps:
+  #     - name: "Check out repository code"
+  #       uses: "actions/checkpout@v3"
+  #     - name: "Setup environment"
+  #       uses: "networktocode/gh-action-setup-poetry-environment@v2"
+  #     - name: "Remove Installed Nautobots (Poetry Workaround)" # Workaround for https://github.com/python-poetry/poetry/issues/2918
+  #       run: "for i in $(ls -d1 ~/.cache/pypoetry/virtualenvs/*/lib/*/site-packages/nautobot-*.dist-info); do poetry run pip uninstall -y nautobot; done"
+  #     - name: "Reinstall Nautobot Local"
+  #       run: "poetry install"
+  #     - name: "Run Integration Tests"
+  #       # If NAUTOBOT_SELENIUM_HOST is set to 'localhost' or '127.0.0.1' the connection does not work
+  #       run: "NAUTOBOT_SELENIUM_HOST=`hostname -f` poetry run invoke integration-test --keepdb"
+  #     - name: "Run coverage"
+  #       run: "poetry run invoke unittest-coverage"
+  #   needs:
+  #     - "pylint"
+  #     - "tests-postgres"
+  #     - "tests-mysql"
 
   container-build:
     name: "Build Container Images (GHCR Only)"
-    runs-on: "ubuntu-20.04"
-    if: "github.event_name == 'push' && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/next')"
-    needs:
-      - "integration-test"
+    runs-on: "ubuntu-22.04"
+    # TBD: Enable the following before review
+    # if: "github.event_name == 'push' && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/next')"
+    # needs:
+    #   - "integration-test"
     strategy:
       fail-fast: true
       matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+        include:
+          - python-version: "3.7"
+            branch: "u/snaselj-4028-ci-integration-workflows"
+            latest: true
+            build-final: true
+          - python-version: "3.8"
+            branch: "u/snaselj-4028-ci-integration-workflows"
+            latest: false
+            build-final: false
+          - python-version: "3.7"
+            branch: "develop"
+            latest: true
+            build-final: false
```

Comment:
We'll probably want a `build` and `publish` flag, per the story: Since there are slight differences between the images, we want to run at least one build of the `final` image but we don't want to publish it. But for the `final-dev` images we want to build and publish them.